system/modules/core/templates/picture/picture_default.html5

```diff
@@ -8,7 +8,9 @@
     <!--[if IE 9]></video><![endif]-->
 <?php endif; ?>
 
-<img src="<?php echo $this->img['src']; ?>"<?php if ($this->img['srcset'] !== $this->img['src']): ?> srcset="<?php echo $this->img['srcset']; ?>"<?php endif; ?><?php if (!empty($this->img['sizes'])): ?> sizes="<?php echo $this->img['sizes']; ?>"<?php elseif (!$this->sources): ?> width="<?php echo $this->img['width']; ?>" height="<?php echo $this->img['height']; ?>"<?php endif; ?> alt="<?php echo $this->alt; ?>"<?php if ($this->title): ?> title="<?php echo $this->title; ?>"<?php endif; ?>>
+<meta itemprop="width" content="<?php echo $this->img['width']; ?>">
```

Comment:
The docs say

> This technique should be used sparingly. Only use meta with content for information that cannot otherwise be marked up.

Can we not mark it up differently?